packages/trader/src/Modules/Account/Sections/Profile/PersonalDetails/personal-details.jsx

```diff
@@ -607,7 +614,7 @@ class PersonalDetailsForm extends React.Component {
                                         has_effect
                                         is_loading={is_btn_loading}
                                         is_submit_success={is_submit_success}
-                                        text={localize('Submit')}
+                                        text={localize('Next')}
```

Comment:
should be Submit? :)

Reply:
it is Next in the design: https://app.zeplin.io/project/5d0b634adaf19b70cdccb972/screen/5d944410e4bb4402686950ff

Reply:
yeah, we've discussed about this with the design team previously. it should be `Submit` instead of `Next` 🙏